deploy/charts/rook-ceph/templates/securityContextConstraints.yaml

```diff
@@ -0,0 +1,91 @@
+# scc for the Rook and Ceph daemons
+# for creating cluster in openshift
+{{- if .Values.openshift }}
+kind: SecurityContextConstraints
+apiVersion: security.openshift.io/v1
+metadata:
+  name: rook-ceph
+allowPrivilegedContainer: true
+allowHostDirVolumePlugin: true
+allowHostPID: false
+# set to true if running rook with host networking enabled
+allowHostNetwork: false
+# set to true if running rook with the provider as host
+allowHostPorts: false
+priority:
+allowedCapabilities: ["MKNOD"]
+allowHostIPC: true
+readOnlyRootFilesystem: false
+# drop all default privileges
+requiredDropCapabilities: ["All"]
+defaultAddCapabilities: []
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: MustRunAs
+fsGroup:
+  type: MustRunAs
+supplementalGroups:
+  type: RunAsAny
+volumes:
+  - configMap
+  - downwardAPI
+  - emptyDir
+  - hostPath
+  - persistentVolumeClaim
+  - projected
+  - secret
+users:
+  # A user needs to be added for each rook service account.
+  - system:serviceaccount:{{ .Release.Namespace }}:rook-ceph-system
+  - system:serviceaccount:{{ .Values.Namespace }}:default
+  - system:serviceaccount:{{ .Values.Namespace }}:rook-ceph-mgr
+  - system:serviceaccount:{{ .Values.Namespace }}:rook-ceph-osd
+  - system:serviceaccount:{{ .Values.Namespace }}:rook-ceph-rgw
```

Comment:
we dont have Namespace in value.yaml right? how does this will get populated or any document to set this value and what it is for?

Reply:
So it is coming from the user input, while we run the `values.yaml`

Reply:
The `.Values.Namespace` would be whatever namespace the chart is being installed to, we don't need to specify a default in values.yaml. Since this is the operator chart, the namespace would be where the operator is installed so I would expect this value to work.